0.6.7/index.html

```diff
@@ -19,6 +20,46 @@
     </script>
     <script type="text/javascript" src="_static/jquery.js"></script>
     <script type="text/javascript" src="_static/doctools.js"></script>
+    <script type="text/javascript">
+    $(function () {
+        $.ajax({
+           url: 'http://docs.sympy.org/releases.txt',
+           success: function(data){
+            var lines = data.split("\n");
+            $.each(lines, function(n, elem) {
+                if (elem != ""){
+                var res = elem.split(":");
+                  if(res[0] == "0.7.6.1"){
+                    $("#version-list").prepend("<li><a style='font-size:100%;' href="+"../latest/index.html"+"><span>Latest</span></a></li>");
+                  }
```

Comment:
@asmeurer Need to update last line in ```releases.txt``` on your server.
```0.7.6.1:SymPy 0.7.6.1``` ---> ```latest:SymPy 0.7.6.1```.in my code currently i set ```0.7.6.1``` take as a latest version if you update this file then i change my condition ```0.7.6.1```--> ```latest```.